### PR TITLE
Documentation about JCasC usage

### LINKED ISSUE #129: Documentation about JCasC usage
**Describe the feature request**
Add documentation with instructions and code snippets how to use [Jenkins Configuration as Code Plugin](https://github.com/jenkinsci/configuration-as-code-plugin) to configure ECU-TEST and TEST-GUIDE settings in [Extensions](https://github.com/jenkinsci/ecutest-plugin#extensions) section in the [README](https://github.com/jenkinsci/ecutest-plugin/blob/master/README.md) file.

### PR DESCRIPTION
**Pull request by cpoenisch**
Resolves #129

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -557,6 +558,231 @@ node('windows') {
...
+    <summary>TEST-GUIDE server</summary>
+
+```yml
+unclassified:
+  testGuide:
+    installations:
+      name: "TEST-GUIDE"
+      toolName: "ECU-TEST"
+      - config:
+          settings:
```
Comment (JanGu-TT) [inline README.md]: These are the missing settings I got also exported:
```
                - atxTextSetting:
                        group: UPLOAD
                        name: "httpProxy"
                  - atxTextSetting:
                        group: UPLOAD
                        name: "httpsProxy"
                - atxBooleanSetting:
                        group: ARCHIVE
                        name: "archiveRecordingMetadata"
                        value: true
                 - atxTextSetting:
                        group: ARCHIVE
                        name: "archiveMiscFilePrefix"
                  - atxTextSetting:
                        group: ARCHIVE
                        name: "archiveFilesPerPackage"
                  - atxBooleanSetting:
                        group: TCF_CONSTANTS
                        name: "mapTcfTesterAsConstant"
                        value: false
                 - atxBooleanSetting:
                        group: SPECIAL
                        name: "includePkgTestSteps"
                        value: true

```

File: README.md [hunk 2]
```diff
@@ -557,6 +558,231 @@ node('windows') {
...
+
+The [Jenkins Configuration as Code Plugin](https://plugins.jenkins.io/configuration-as-code) (a.k.a. JCasC) allows to configure Jenkins based on human-readable declarative configuration files. This plugin supports the external configuration of ECU-TEST tool installations and of TEST-GUIDE server instances in the Jenkins global configuration.
+
+Existing configurations of both types can be exported to according YAML file using the [JCasC export view](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/configExport.md).
+
+<details>
+    <summary>ECU-TEST installation</summary>
+
+```yml
+tool:
```
[the remark this answers is not in the document]
Comment (cpoenisch) [reply README.md]: Order and indent does not matter here.

File: README.md (hunk 1 above)
Comment (cpoenisch) [reply README.md]: Used recent settings of TEST-GUIDE 1.76

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JanGu-TT) [reply README.md]: I agree that order does not matter, but indent does by yaml format. Fixed with commit.

File: README.md [hunk 3]
```diff
@@ -557,6 +558,262 @@ node('windows') {
...
+</details>
+
+<details>
+    <summary>TEST-GUIDE server</summary>
+
+```yml
+unclassified:
+  testGuide:
+    installations:
+    name: "TEST-GUIDE"
```
Comment (JanGu-TT) [inline README.md]: `name` & `toolName` are  still under installations key not as in the exported configuration unde config. Bug in exporter or bug in yaml file?:
```
   installations:
        - config:
              name: "TEST-GUIDE"
              toolName: "ECU-TEST 8.0"
              settings:
                  - atxBooleanSetting:
```

Comment (cpoenisch) [reply README.md]: Fixed

**Pull request merged**